box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -100,15 +122,15 @@ private void setSampleFolder(final BoxFolder folder){
             public void run() {
                 mSampleFolder = folder;
                 if (mSampleFolder == null){
-                    findViewById(R.id.launchShare).setVisibility(View.GONE);
-                    findViewById(R.id.launchCollabs).setVisibility(View.GONE);
-                    findViewById(R.id.removeSampleFolder).setVisibility(View.GONE);
-                    findViewById(R.id.createSampleFolder).setVisibility(View.VISIBLE);
+                    mShareBtn.setVisibility(View.GONE);
+                    mCollabsBtn.setVisibility(View.GONE);
+                    mChooseActionTv.setVisibility(View.GONE);
```

Comment:
Probably change the layout to group these in one parent - so you can just toggle the visibility of the container.